builds/azure-pipelines-release.yml

```
@@ -26,12 +26,22 @@ jobs:
       script: |
         [xml]$xmlDoc = Get-Content $(Build.SourcesDirectory)\src\Files.Package\Package.appxmanifest
         $xmlDoc.Package.Identity.Name="Files"
-        $xmlDoc.Package.Identity.Publisher="CN=53EC4384-7F5B-4CF6-8C23-513FFE9D1AB7"
+        $xmlDoc.Package.Identity.Publisher="Files"
         $xmlDoc.Package.Properties.DisplayName="Files"
         $xmlDoc.Package.Applications.Application.VisualElements.DisplayName="Files"
         $xmlDoc.Save('$(Build.SourcesDirectory)\src\Files.Package\Package.appxmanifest')
       failOnStderr: true
 
+  - task: PowerShell@2
+    displayName: 'Use Release Logo'
+    inputs:
+      targetType: 'inline'
+      script: |
+        gci $(Build.SourcesDirectory)\src -Include *.csproj, *.appxmanifest, *.wapproj -recurse | ForEach -Process {
+        (Get-Content $_ -Raw | ForEach -Process {$_ -replace "Assets\AppTilesDev", "Assets\AppTiles"}) | Set-Content $_ -NoNewline
```

Regarding my previous comment, this is the thing that I had to change in order to work. (1/3)
```suggestion
        (Get-Content $_ -Raw | ForEach -Process {$_ -replace "Assets\\AppTilesDev", "Assets\AppTiles"}) | Set-Content $_ -NoNewline
```